linera-core/src/client/mod.rs

```diff
@@ -422,6 +423,54 @@ where
     }
 }
 
+// Tracks which chain_id<>block_height pairs have been successfully downloaded.
```

Comment:
```suggestion
/// Tracks which chain_id<>block_height pairs have been successfully downloaded.
```

Reply:
(Also other comments that could be doc comments.)

Reply:
Also, maybe `` `(ChainId, BlockHeight)` pairs`` or `chain ID-block height pairs`?